test/testdata/cfg/examples.rb.flatten-tree.exp

```diff
@@ -1,9 +1,9 @@
 begin
-  <emptyTree>
+  ::<Magic>.<define-top-class-or-module>(::<root>)
```

Comment:
I could proactively skip emitting this, but I think it's fine to be here as it won't be traversed by CFG.

Reply:
let's skip it, makes us more regular